pom.xml

```diff
@@ -92,6 +93,29 @@
 
     <build>
         <plugins>
+            <plugin>
+                <artifactId>maven-jar-plugin</artifactId>
+                <version>2.6</version>
+                <configuration>
+                    <archive>
+                        <manifest>
+                            <addClasspath>true</addClasspath>
```

Comment:
We don't want to add the classpath for this project. You only add the classpath to projects that have a `main` function (i.e. are executable), this project is a library.